src/main/java/org/gradle/profiler/ScenarioLoader.java

```diff
@@ -95,6 +89,7 @@ class ScenarioLoader {
     private static final String TOOLING_API = "tooling-api";
     private static final String ANDROID_STUDIO_SYNC = "android-studio-sync";
     private static final String ANDROID_STUDIO_JVM_ARGS = "studio-jvm-args";
+    private static final String ANDROID_STUDIO_IDEA_PROPERTIES = "idea-properties";
```

Comment:
:nail_care: Could we also update [an example](https://github.com/gradle/gradle-profiler/blob/5dd7dce5cbc9e838ab2054bf3b4f01780a020f91/README.md?plain=1#L280-L283) in README.md and add also a comment what it does.

It might help if we just add example with the configuration we will use (is it `gradle.tooling.models.parallel.fetch=true`?)